### PR TITLE
prevent TAG designating targets for hostile homing munitions

### PR DESCRIPTION
**Pull request by NickAragua**
Pretty straightforward - apparently, you could use TAG to designate targets for hostile Arrow IVs and such. Didn't seem correct, so.

### CODE REVIEW

File: megamek/src/megamek/common/weapons/ArtilleryWeaponIndirectHomingHandler.java [hunk 1]
```diff
@@ -331,8 +331,16 @@ public void convertHomingShotToEntityTarget() {
 
         Vector<TagInfo> v = game.getTagInfo();
         Vector<TagInfo> allowed = new Vector<TagInfo>();
+        Entity attacker = game.getEntityFromAllSources(getAttackerId());
+        
         // get only TagInfo on the same side
         for (TagInfo ti : v) {
+            // let's don't allow
```
Comment (Windchild292) [inline megamek/src/megamek/common/weapons/ArtilleryWeaponIndirectHomingHandler.java:338]: Should finish this comment

**Review by Windchild292 — COMMENTED** [review]
Code looks good outside of an unfinished comment, waiting on rules check

File: megamek/src/megamek/common/weapons/ArtilleryWeaponIndirectHomingHandler.java (hunk 1 above)
Comment (NickAragua) [reply megamek/src/megamek/common/weapons/ArtilleryWeaponIndirectHomingHandler.java]: It was redundant anyway, got rid of it.

**Review by Windchild292 — APPROVED** [review]
Code is good, but should be pending rules check

**Comment by NickAragua** [comment]
What rules check?

**Comment by Windchild292** [comment]
You asked if it was intentional in the Slack and never got a solid answer?

**Review by Windchild292 — APPROVED** [review]

**Pull request merged**